Packs/Lastline/Integrations/Lastline_v2/Lastline_v2.py

```diff
@@ -312,6 +313,7 @@ def main():
         'api_token': params.get('api_token')
     }
     api_params.update(demisto.args())
+    threshold = int(params.get('threshold', 70))
```

Comment:
This threshold is still taken from the params isn't it?
I suggest:
threshold = int(api_params.get('threshold, params.get('threshold', 70)))

Reply:
<img width="359" alt="Screen Shot 2021-01-20 at 11 33 20" src="https://user-images.githubusercontent.com/57587340/105155492-55fbbe80-5b13-11eb-9d91-c0605b33fa51.png">